src/targets/services/softDeleteOrRestoreAccounts.js

```diff
@@ -0,0 +1,40 @@
+import NodeVaultClient from 'cozy-keys-lib/transpiled/NodeVaultClient'
+import CozyClient from 'cozy-client'
+import logger from 'cozy-logger'
+import polyfillFetch from './polyfillFetch'
+
+import softDeleteOrRestoreAccounts from 'cozy-harvest-lib/dist/services/softDeleteOrRestoreAccounts'
+
+const log = logger.namespace('softDeleteOrRestoreAccounts')
+
+polyfillFetch()
+
+const main = async () => {
+  const vaultClient = new NodeVaultClient(process.env.COZY_URL)
+
+  const cozyClient = CozyClient.fromEnv()
+
+  try {
+    await softDeleteOrRestoreAccounts(
+      cozyClient,
+      vaultClient,
+      JSON.parse(process.env.COZY_COUCH_DOC)
+    )
+  } catch (err) {
+    if (err.message === 'DECRYPT_FAILED') {
```

Comment:
To share the error with harvest, it'd be good to export the constant from there, and import it here. It would show in Harvest that it's expected that outside code rely on the error.

Reply:
Sounds like a good idea, especially that several services are impacted. I made an issue https://github.com/cozy/cozy-libs/issues/1108